class/UserAccessManager.php

```diff
@@ -953,38 +960,63 @@ public function endsWith($sHaystack, $sNeedle)
      */
     
     /**
-     * The function for the wp_print_styles action.
+     * Register styles and scripts with handle for admin panel.
      */
-    public function addStyles()
-    {
-        wp_enqueue_style(
-            'UserAccessManagerAdmin', 
-            UAM_URLPATH . "css/uamAdmin.css",
+    protected function registerAdminStylesAndScripts() {
+        wp_register_style(
+            self::HANDLE_STYLE_ADMIN,
+            UAM_URLPATH . 'css/uamAdmin.css',
             array() ,
-            '1.0',
+            $this->_sUamVersion,
             'screen'
         );
-        
-        wp_enqueue_style(
-            'UserAccessManagerLoginForm', 
-            UAM_URLPATH . "css/uamLoginForm.css",
-            array() ,
-            '1.0',
-            'screen'
+        wp_register_script(
+            self::HANDLE_SCRIPT_ADMIN,
+            UAM_URLPATH . 'js/functions.js',
+            array('jquery'),
+            $this->_sUamVersion
         );
     }
-    
+
     /**
-     * The function for the wp_print_scripts action.
+     * The function for the admin_enqueue_scripts action for styles and scripts.
+     * @param string $hook 
      */
-    public function addScripts()
+    public function enqueueAdminStylesAndScripts($hook)
```

Comment:
The hungarian notation (see [https://en.wikipedia.org/wiki/Hungarian_notation](https://en.wikipedia.org/wiki/Hungarian_notation)) is used for uam so it should be  `$sHook` instead of `$hook`